### ISSUE TITLE
[Feature] Unable to access Cloud Code via local IP

### ISSUE THREAD

**Issue by stx** [GoogleCloudPlatform/cloud-code-vscode#444]
Cloud Code only appears to bind on localhost, preventing access via local IP (and, say, your phone, or any other debugging or development device.)

Is there a way to fix this or have it bind on 0.0.0.0?

**Comment by sivakku** [comment]
Ok, thanks for the feedback, you would need to add address field in the skaffold.yaml  for this to configure to any other host IP as well.

https://skaffold.dev/docs/pipeline-stages/port-forwarding/

```
portForward:
- resourceType: deployment
  resourceName: myDep
  namespace: mynamespace
  port: 8080
  **address: 0.0.0.0**
  localPort: 9000
```

**Comment by stx** [comment]
@sivakku We're using Cloud Run which doesn't appear to support skaffoldConfig in launch.json. How would you suggest making this work using the Cloud Run tool?

**Comment by sivakku** [comment]
> @sivakku We're using Cloud Run which doesn't appear to support skaffoldConfig in launch.json. How would you suggest making this work using the Cloud Run tool?

Missed minor detail. For Cloud Run we auto generate configuration and is no way to specify additional skaffold configuration. i have to mark this as a feature request. is the scenario is to test/develop the cloud run service that is consumed by mobile apps?

**Comment by stx** [comment]
@sivakku Exactly.

**Comment by sivakku** [comment]
With that config, it might not work if the mobile and local laptop used for testing are not on the same user NIC. The underneath is just a call to kubectl port forward. Can you verify if it works through kubectl by running it from terminal?

https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#port-forward

```kubectl port-forward --address localhost,10.19.21.23 pod/mypod 8888:5000```

**Comment by j-windsor** [comment]
This will be fixed in our next release towards the end of this month. Thanks!

**Comment by j-windsor** [comment]
This has been fixed in Cloud Code v1.14.0. Thanks for reporting!
